pkg/odo/cli/pipelines/init.go

```diff
@@ -92,9 +92,9 @@ func NewCmdInit(name, fullName string) *cobra.Command {
 		},
 	}
 
-	initCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "GitOps repository e.g. https://github.com/organisation/repository")
+	initCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "provide a GitOps repository(it could be GitHub or GitLab) e.g. https://github.com/organisation/repository")
 	initCmd.MarkFlagRequired("gitops-repo-url")
-	initCmd.Flags().StringVar(&o.GitOpsWebhookSecret, "gitops-webhook-secret", "", "provide the GitHub webhook secret for GitOps repository (if not provided, it will be auto-generated)")
+	initCmd.Flags().StringVar(&o.GitOpsWebhookSecret, "gitops-webhook-secret", "", "provide the GitHub/GitLab webhook secret for GitOps repository (if not provided, it will be auto-generated)")
```

Comment:
I don't think list out all the supported Git services is the way to go.  Why not using something like below thatt you use for bootstrap command?
"provide a secret that we can use to authenticate incoming hooks from your Git hosting service for GitOps repository (if not provide ..."